inject-java/src/main/java/io/micronaut/annotation/processing/AbstractInjectAnnotationProcessor.java

```diff
@@ -63,6 +79,28 @@ public SourceVersion getSupportedSourceVersion() {
         }
     }
 
+    @Override
+    public Set<String> getSupportedOptions() {
+        final Set<String> options = CollectionUtils.setOf("org.gradle.annotation.processing.aggregating");
```

Comment:
For consistency, this should probably also be guarded by `isIncremental`.

Reply:
Good point, will make the change